pkg/filesystem/local_directory_windows.go

```diff
@@ -108,11 +109,7 @@ func newLocalDirectory(absPath string, openReparsePoint bool) (DirectoryCloser,
 }
 
 func NewLocalDirectory(path string) (DirectoryCloser, error) {
```

Comment:
Could we please change `NewLocalDirectory()` now to take a `path.Parser` instead of a string now?

Reply:
Yes! I have the corresponding changes to bb-remote-execution on the way, and it improved the build working directory code I had outstanding before, which is nice.

Reply:
https://github.com/buildbarn/bb-remote-execution/pull/138